packages/commonwealth/client/scripts/views/components/proposals/proposals_explainers.tsx

```diff
@@ -72,7 +72,7 @@ export const CompoundProposalStats = (props: CompoundProposalStatsProps) => {
       statAction={
         <CWButton
           buttonType="primary-blue"
-          onClick={() => navigate('/new/proposal')}
+          onClick={() => navigate('/new/proposal', { deferChain: false })}
```

Comment:
everywhere you use navigate with added `deferChain` param in this PR => this does not do anything. Second argument is used by react-router ([see here](https://reactrouter.com/en/6.10.0/hooks/use-navigate)) and it should only accept `replace`, `state`, and `relative`. You can check this putting the `  console.log('params.deferChain', params.deferChain)
` in `LayoutWrapper`. Does not matter what you pass to navigate, it will always display the value defined in `commonDomainRoutes`. Does it make sense?